### PR TITLE
Adding reloading to existing view models

### PR DESCRIPTION
**Pull request by nickrandolph**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Bug fix

### :arrow_heading_down: What is the current behavior?
Reload isn't called on resuming from suspension

### :new: What is the new behavior (if this is a feature change)?
Reload is called

### :bug: Recommendations for testing
Run playground UWP sample

### :memo: Links to relevant issues/docs
https://github.com/MvvmCross/MvvmCross/issues/2388

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [x] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [x] Nuspec files were updated (when applicable)
- [x] Rebased onto current develop

Note: Please take a look at the calling of ReloadState in the MvxWindowsViewContainer as I'm not 100% sure whether this will cause other issues

### CODE REVIEW

File: TestProjects/Playground/Playground.Uwp/App.xaml.cs [hunk 1]
```diff
@@ -53,11 +57,14 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             {
                 if (rootFrame.Content == null)
                 {
-                    var setup = new Setup(rootFrame);
+                    var setup = new Setup(rootFrame,"Suspension");
```
Comment (Cheesebaron) [inline TestProjects/Playground/Playground.Uwp/App.xaml.cs]: add space

File: TestProjects/Playground/Playground.Uwp/App.xaml.cs [hunk 2]
```diff
@@ -81,11 +88,23 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
...
             deferral.Complete();
         }
+
+        private async void OnResuming(object sender, object e)
+        {
+            var suspension = Mvx.GetSingleton<IMvxSuspensionManager>() as MvxSuspensionManager;
+
+            await suspension.RestoreAsync();
+        }
+
```
Comment (Cheesebaron) [inline TestProjects/Playground/Playground.Uwp/App.xaml.cs]: Remove trailing newline

File: TestProjects/Playground/Playground.Uwp/App.xaml.cs [hunk 3]
```diff
@@ -53,11 +57,14 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             {
                 if (rootFrame.Content == null)
                 {
-                    var setup = new Setup(rootFrame);
+                    var setup = new Setup(rootFrame,"Suspension");
                     setup.Initialize();
 
                     var start = MvvmCross.Platform.Mvx.Resolve<MvvmCross.Core.ViewModels.IMvxAppStart>();
                     start.Start();
+
```
Comment (Cheesebaron) [inline TestProjects/Playground/Playground.Uwp/App.xaml.cs]: Remove extra newlines

**Comment by nickrandolph** [comment]
@Cheesebaron happy to fix up the styling issues. The code factor issue was because there were two classes in the same file - I can split them out but it was like that before this PR (let me know what you'd prefer me to do with this one...).

Also, any thoughts on the call to ReloadState I added - are there cases where this would cause bad side-effects?

File: TestProjects/Playground/Playground.Uwp/App.xaml.cs (hunk 1 above)
Comment (nickrandolph) [reply TestProjects/Playground/Playground.Uwp/App.xaml.cs]: done

File: TestProjects/Playground/Playground.Uwp/App.xaml.cs (hunk 3 above)
Comment (nickrandolph) [reply TestProjects/Playground/Playground.Uwp/App.xaml.cs]: done

File: TestProjects/Playground/Playground.Uwp/App.xaml.cs (hunk 2 above)
Comment (nickrandolph) [reply TestProjects/Playground/Playground.Uwp/App.xaml.cs]: done

**Pull request merged**